### PR TITLE
Added feature docs for text fragment language.

### LINKED ISSUE #9211: Document text part language feature
## 📝 Provide a description of requested docs changes

Followup ticket for text part feature documentation - https://github.com/ckeditor/ckeditor5/pull/9074

### PR DESCRIPTION
**Pull request by jacekbogdanski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Docs (language): Added feature documentation for text fragment language. Closes #9211.

---

### Additional information

For now, I've just added separate docs for `Language` entry, however, it may not be the best fit, see https://github.com/ckeditor/ckeditor5/issues/9211#issuecomment-801073036

Some links are failing during the build:
image
I'm not really sure why it happens as the same links in API docs are working correctly - would be glad for some help with that.

### CODE REVIEW

File: packages/ckeditor5-language/docs/features/language.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+// Applies language to the selected text part with the given language code.
+editor.execute( 'textPartLanguage', { languageCode: 'es' } );
+
+// Optionally, you can also provide text direction information ('ltr' or 'rtl'),
+// however, note that the feature will resolve text direction by itself, so in most
+// cases this option is redundant.
+editor.execute( 'textPartLanguage', {
+	languageCode: 'ar',
+	textDirection: 'rtl' // Optional text direction.
+} );
```
Comment (jacekbogdanski) [inline packages/ckeditor5-language/docs/features/language.md:73]: Not sure if it makes sense to mention the optional `textDirection` option.

Comment (godai78) [reply packages/ckeditor5-language/docs/features/language.md:72]: I'd say let's drop it, if not for the "most cases". If it has some specific use, we may want to mention the specific use in docs.

**Review by AnnaTomanek — APPROVED** [review]

File: docs/features/spelling-and-grammar-checking.md [hunk 2]
```diff
@@ -33,6 +33,12 @@ By default the spell checker supports 18 languages: AI-based English (default),
 
 There are also over 150 additional languages and specialized dictionaries such as medical and legal available for an additional fee. You can check the full list [here](https://webspellchecker.com/additional-dictionaries/).
 
+## Related features
+
+There are other language-related CKEditor 5 features you may want to check:
+
+* {@link features/language Text part Language}  &ndash; Set the language of selected content part.
```
Comment (AnnaTomanek) [inline docs/features/spelling-and-grammar-checking.md:40]:
```suggestion
* {@link features/language Text part Language}  &ndash; Set the language of the selected content part to support multilingual texts.
```

File: docs/features/ui-language.md [hunk 3]
```diff
@@ -191,3 +191,9 @@ ClassicEditor
 <info-box>
 	If you are unsure about the language that the content will be typed in, do not set it. The language of the content will then be inherited from the {@link module:core/editor/editorconfig~EditorConfig#language language of the UI}.
 </info-box>
+
+## Related features
+
+There are other CKEditor 5 features that will help you control the content language:
+
+* {@link features/language Text part Language}  &ndash; Set the language of selected content part.
```
Comment (AnnaTomanek) [inline docs/features/ui-language.md:199]:
```suggestion
* {@link features/language Text part Language}  &ndash; Set the language of the selected content part to support multilingual texts.
```

**Pull request merged**